test/src/test/java/org/corfudb/runtime/clients/TestClientRouter.java

```diff
@@ -188,6 +194,13 @@ public IClientRouter addClient(IClient client) {
                                                                     .port(port).build());
         }
 
+        // Set up the timer and context to measure request
+        final String timerName = CorfuComponent.CR.toString() + message.getMsgType().name().toLowerCase();
```

Comment:
Same here

Reply:
Done. (Adding a map to avoid re-construction)